lib/tasks/emails.rake

```diff
@@ -0,0 +1,10 @@
+namespace :emails do
+
+  desc 'send reminder email'
+  task reminder_email: :environment do
+    User.all.each do |user|
```

Comment:
on the off chance we ever get new users, this should not break

Reply:
What is controlling how often these emails get sent?

Reply:
my plan was to use the free heroku add on (scheduler)[https://devcenter.heroku.com/articles/scheduler] to run the rake task once every 24 hours. Way easier than installing redis and sidekiq or dealing with whenever with a chron tab to do essentially the same exact thing.

Reply:
Ok. Does it trigger anytime in localhost? Want to make sure we don't get spammed with emails lol

Reply:
No, to get it to trigger you need to run rake email:reminder_email.

Reply:
Cool cool